app/controllers/users_controller.rb

```diff
@@ -0,0 +1,14 @@
+class UsersController < ApplicationController
+
+  def show
+    hash =  {
+      'email' => session['email'],
+      'first_name' => session['first_name'],
+      'last_name' => session['last_name']
+      }
+
+    @recipes = UserFacade.get_user_recipes(hash)
+    require "pry"; binding.pry
```

Comment:
@dylan-harper can you remove this pry? probably don't want to merge it with this there...

Reply:
Can do!